README.md

```diff
@@ -11,6 +11,8 @@ I built this website with a mobile-first approach, applying fluid design princip
 
 Check out my video presentation [here](https://www.loom.com/share/33182d5f32f2450598a2c0cc4212f1e2)
```

Comment:
- [ ] Please in the presentation video you should highlight a piece of the code you find interesting or proud of making.

Reply:
Thank you for the review, I made the changes requested.